### PR TITLE
Upgrade to modern patterns

### PR DESCRIPTION
**Pull request by RonaldEAM**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -31,45 +29,23 @@ jobs:
...
-        run: |
-          echo "//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}" > .npmrc
-          yarn
-          npm publish
+          token: ${{ secrets.AUTO_GITHUB_PAT_TOKEN }}
+      - name: Build and Release
+        uses: jupiterone/action-npm-build-release@v1
+        with:
+          npm_auth_token: ${{ secrets.NPM_AUTH_TOKEN }}
+          gh_token: ${{ secrets.AUTO_GITHUB_PAT_TOKEN }}
```
Comment (dobregons) [inline .github/workflows/build.yml:51]: Is this repo already configured with `auto`??

**Review by dobregons — COMMENTED** [review]
Also, please take a look at the `build` step, it's failing https://github.com/JupiterOne/graph-snipe-it/actions/runs/4318980353/jobs/7538015504

File: .github/workflows/build.yml (hunk 1 above)
Comment (VDubber) [reply .github/workflows/build.yml:51]: I don't think I've requested that change. @RonaldEAM could you create a DESK ticket like this [one](https://jupiterone.atlassian.net/browse/DESK-2479) for snipe IT?

Comment (RonaldEAM) [reply .github/workflows/build.yml:51]: Sure, I will

Comment (RonaldEAM) [reply .github/workflows/build.yml:51]: Ticket created: https://jupiterone.atlassian.net/browse/DESK-3198

**Review by VDubber — APPROVED** [review]

**Pull request merged**